common/src/main/java/com/revenuecat/purchases/common/logWrapper.kt

```diff
@@ -67,5 +68,9 @@ enum class LogIntent(val emojiList: List<String>) {
     /**
      * Emojis for Amazon info messages.
      */
-    AMAZON_WARNING(listOf(Emojis.BOX, Emojis.DOUBLE_EXCLAMATION))
+    AMAZON_WARNING(listOf(Emojis.BOX, Emojis.DOUBLE_EXCLAMATION)),
```

Comment:
there's a disconnect here between the coment and the code - "Amazon Info" vs `AMAZON_WARNING`. 
Also, are these comments needed at all? I feel like the comment at the start of the class is already sufficient to understand what each of these are. I realize that this was here before, so maybe we can keep it for this PR